extra/catena-message-0x16-format.md

```diff
@@ -0,0 +1,528 @@
+# Understanding Catena 4450 format 0x16
```

Comment:
This doc also describes 0x14. The correct thing is to add this to the other md, rather than creating a new one.

Reply:
You must also update all the .js files to understand 0x16.

Reply:
Ok, I will merge the 0x14 format md file and 0x16 format md file. Also try to update js files.